src/entry.cpp

```diff
@@ -153,6 +153,6 @@ int main(void)
 
         t += 0.01f;
     }
-
+    window->~Window();
```

Comment:
Don't call the destructor, this doesn't actually free the memory, and it shouldn't be called directly usually.
Instead use `delete window;` (its like a free() that calls the destructor).